components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/editor/commons/lib/ace-editor/mode/asyncapi.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2014, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
add correct license header and check other places as well

Reply:
fixed with https://github.com/wso2/carbon-analytics/pull/1913/commits/d807fbc380a7203f1238bcad5425d84778a91878